src/components/Onfido/BaseOnfidoWeb.tsx

```diff
@@ -78,17 +81,15 @@ function initializeOnfido({sdkToken, onSuccess, onError, onUserExit, preferredLo
                 },
             },
         ],
-        smsNumberCountryCode: CONST.ONFIDO.SMS_NUMBER_COUNTRY_CODE.US,
```

Comment:
We don't have `CONST.ONFIDO.SMS_NUMBER_COUNTRY_CODE.US`, we have only `CONST.ONFIDO.SMS_NUMBER_COUNTRY_CODE`, so all this time the `undefined` was used

Reply:
and what about `showCountrySelection`?

Reply:
This comment is about `showCountrySelection` https://github.com/Expensify/App/pull/31378#discussion_r1394420768

Reply:
ohh right, thanks!